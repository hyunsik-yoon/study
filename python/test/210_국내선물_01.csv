거래일자,거래번호,거래종류,적요명,종목번호,종목명,수량,단가,거래금액,수수료,예수금전잔,예수금금잔,정산금액,약정금액,세금합계,대용잔고,기타수표금액,미수/변제,대체금융회사,계좌명,받는분 표시내용,보내는분 표시내용
2023-03-06,4,매도,당일차금매도출금                                                                ,1B3T3000,고려아연   F 202303 (  10)                                                                                                                                                                                                                                                                                  ,50,"592,000","500,000","28,620","309,682,205","309,153,585","528,620","296,000,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-03-10,15,매수,당일차금매수출금                                                                ,1B3T3000,고려아연   F 202303 (  10)                                                                                                                                                                                                                                                                                  ,50,"593,000",0,"28,670","307,652,065","307,623,395","28,670","296,500,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
