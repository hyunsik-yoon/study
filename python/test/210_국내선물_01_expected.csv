종목명,진입일자,진입단가,청산일자,청산단가,거래수량,총수수료,총세금합계,순수익
고려아연   F 202303 (  10),03/06/2023,592000,03/10/2023,593000,50,57290,0,-500000
